RegistrationView.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import './RegistrationView.css';
+
+const axios = require('axios');
+
+class RegistrationView extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            login: '',
+            email: '',
+            password: ''
+        };
+        this.handleLoginChange = this.handleLoginChange.bind(this);
+        this.handleEmailChange = this.handleEmailChange.bind(this);
+        this.handlePasswordChange = this.handlePasswordChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+    }
+
+    handleSubmit(event){
+        event.preventDefault();
+        console.log('form is submitted. Login:', this.state.login,
+            'Email:', this.state.email,
+            'Password:', this.state.password);
+
+        let addUser = {                      //let login, email, pass from forms
+            login: this.state.login,
+            email: this.state.email,
+            password: this.state.password
+        };
+        axios.post('../../../server/RegistrationForm', addUser )    //Send Login, email, pass to server
+
+        axios.get('http://localhost:3000/register')                //Check HTTP status code

```

Comment:
Unnecessary GET request, HTTP status code should be checked in POST request

Reply:
generally do not hardcode URI in application http://localhost:3000/ 
It should be passed in npm run parameter.